### PR TITLE
Update Shu Yang's information in AboutUs page

### LINKED ISSUE #19: Update Shu Yang's information in aboutUs page

### PR DESCRIPTION
**Pull request by shuyangk**

### CODE REVIEW

File: docs/AboutUs.md [hunk 1]
```diff
@@ -19,15 +19,15 @@ You can reach us at the email `seer[at]comp.nus.edu.sg`
 
 * Role: Project Advisor
 
-### Jane Doe
+### Shu Yang
```
Comment (yytan25) [inline docs/AboutUs.md:22]: LGTM!

**Review by tanyyyming — APPROVED** [review]
LGTM!

File: docs/AboutUs.md (hunk 1 above)
Comment (tanyyyming) [reply docs/AboutUs.md:22]: If it's good to go, you can approve it!

**Review by yytan25 — APPROVED** [review]

File: docs/AboutUs.md (hunk 1 above)
Comment (yytan25) [reply docs/AboutUs.md:22]: Approved!

**Pull request merged**